### PR TITLE
Rerun last test

### PR DESCRIPTION
**Pull request by timonschenzel**
Nice feature for rerun your last test from any other file within your project.

### CODE REVIEW

**Comment by lorisleiva** [comment]
I was specifically looking for that feature. That would be amazing.

File: sublime-phpunit.sublime-commands [hunk 1]
```diff
@@ -2,5 +2,6 @@
     { "caption": "Sublime PHPUnit: Run This File", "command": "run_phpunit_test" },
     { "caption": "Sublime PHPUnit: Run This Directory", "command": "run_phpunit_tests_in_dir" },
     { "caption": "Sublime PHPUnit: Run This Test Method", "command": "run_single_phpunit_test" },
+    { "caption": "Sublime PHPUnit: Run Last Test", "command": "run_last_phpunit_test" },
     { "caption": "Sublime PHPUnit: Run All Tests", "command": "run_all_phpunit_tests" }
-]
+]
```
Comment (DCzajkowski) [inline sublime-phpunit.sublime-commands:7]: The new line at the end of a file would be appreciated :)

File: sublime-phpunit.py [hunk 2]
```diff
@@ -98,6 +102,18 @@ def run(self, *args, **kwargs):
...
 
+class RunLastPhpunitTestCommand(PhpunitTestCommand):
+
+    def run(self, *args, **kwargs):
+        file_name, phpunit_config_path, phpunit_bin, active_view, directory = self.get_paths()
+
+        current_function = self.get_current_function(active_view)
+
+        if 'Test' in file_name:
+            self.run_in_terminal('cd ' + phpunit_config_path + ' && phpunit ' + file_name + " --filter '/::" + current_function + "$/'")
```
Comment (DCzajkowski) [inline sublime-phpunit.py]: Can you reuse existing implementation, ie run the method inside the `RunSinglePhpunitTestCommand` class? Maybe extend it and call the parent's `run()` method?

**Review by DCzajkowski — CHANGES_REQUESTED** [review]
I wanted to make a PR like this myself. It's awesome you have made it! Thanks and I hope it will be merged very soon. 😃

File: sublime-phpunit.py (hunk 2 above)
Comment (timonschenzel) [reply sublime-phpunit.py]: Good suggestion, I have reused existing implementation.

Comment (DCzajkowski) [reply sublime-phpunit.py]: Awesome! I hope it will be merged very soon :)

**Comment by DCzajkowski** [comment]
Woohoo! 🎉

**Comment by timonschenzel** [comment]
Nice! :smile:

**Pull request merged**